command/certificate/remote_test.go

```diff
@@ -66,7 +66,7 @@ func TestGetPeerCertificateServerName(t *testing.T) {
 	tests := map[string]newTest{
 		"sni-disabled-host": {host, "", nil},
 		"sni-enabled-host":  {host, serverName, nil},
-		"sni-disabled-ip":   {addr, "", fmt.Errorf("failed to connect: x509: cannot validate certificate for %s because it doesn't contain any IP SANs", addr)},
+		"sni-disabled-ip":   {addr, "", fmt.Errorf("failed to connect")},
```

Comment:
not quite sure why this is included in the PR - in case it's not a mistake, maybe use `errors.New` instead of `fmt.Errorf` as there's no interpolation taking place.

Reply:
This fixed a problem with the unit tests, I don't generally compare error strings, but this test was doing this, and it was failing on 1.18 because the string is now different, I've just removed the course, so it will match only with the wrapped string.

Reply:
I changed to use `errors` with e599b7c